app/main.py

```diff
@@ -1,3 +1,55 @@
+from typing import Union
+import math
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
```

Comment:
What if user passes wrong data type to our class? Should we have some validation in place?

Reply:
I see you haven't started topic with properties and descriptors yet. So if you can't do it right now you can ignore this comment. But if you can do it you will be amazing :)